src/components/AddTokenButton/AddTokenButton.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState, useMemo } from 'react'
+import styled from 'styled-components'
+import { SWPR } from '@swapr/sdk'
+import { useActiveWeb3React } from '../../hooks'
+import { useTokenBalance } from '../../state/wallet/hooks'
+
+const MAINNET_ADDRESS = '0x6cacdb97e3fc8136805a9e7c342d866ab77d0957'
```

Comment:
I'd import these from the SDK. It's important to have a single source of truth for these addresses.